plugins/feeds/public/otx_alienvault.py

```diff
@@ -1,11 +1,13 @@
 import logging
+from datetime import datetime
```

Comment:
You may want to use a single statement to import the objects you need  from `datetime`:
`from datetime import datetime, timedelta`.

Reply:
Yeah. I think some linters may be unhappy with that but if it doesn't affect (human) readability too much...

Reply:
I don't think that would be a problem for linters (I'd even say the opposite). According to PEP8, this is bad:
```Python
import os, sys
```

... but this is fine:
```Python
from sys import stdout, stderr
```